### ISSUE TITLE
Mirador accessibility test configurations for DZB Lesen / BIKOSAX report

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2936]
Goal is to set up 5-7 configurations that represent the basic functionality of Mirador, with direct URL access to the configuration so it can be used for both automated and manual accessibility testing. Each configuration requires one or more manifests to illustrate the functionality in a meaningful way. I've included example manifests; if someone has a better illustration for any of these tasks, please feel free to suggest it.

# 1. Start
<img width="500" alt="start" src="https://user-images.githubusercontent.com/6945691/73987136-44b2a580-48f4-11ea-9f2c-e1e9f7c9ad56.png">

**Tasks**: Assess opening screen; open catalog

# 2. Catalog
<img width="500" alt="catalog" src="https://user-images.githubusercontent.com/6945691/73987191-66139180-48f4-11ea-82fc-99eca9357161.png">

**Tasks**: Add a resource; open a resource

**Manifests**: Catalog should not include resources that cannot be opened (e.g. "The biological basis of medicine"). Probably should include all the manifests included in the testing?

# 3. Two-up comparison
<img width="500" alt="2-up mss" src="https://user-images.githubusercontent.com/6945691/73987267-a115c500-48f4-11ea-8b09-50bb886fc347.png">

**Tasks**: Open a second resource.

**Manifests**: Two related manifests with metadata and meaningful canvas labels.

- Civitates Orbis Terrarum.
https://purl.stanford.edu/db076zm5895/iiif/manifest
- Cambridge, Corpus Christi College, MS 298: Laurence Wade OSB, Life of Becket. Canterbury Documents
https://purl.stanford.edu/np593hr6132/iiif/manifest

### 3.1 Metadata
<img width="500" alt="metadata mss" src="https://user-images.githubusercontent.com/6945691/73987289-b7238580-48f4-11ea-9b8c-e597eabf8e48.png">

**Task**: Read/compare metadata of two manuscripts. Requires using the toggle to open the sidebar in both windows.

### 3.2 Navigation
<img width="500" alt="navigate mss" src="https://user-images.githubusercontent.com/6945691/73987296-bbe83980-48f4-11ea-939c-366aa47d7990.png">

**Tasks**: Use the index to navigate to different pages in the manuscripts. Requires using the sidebar menu to select the index for both manuscripts. 

# 4. Annotations
<img width="500" alt="annotations" src="https://user-images.githubusercontent.com/6945691/73987349-ec2fd800-48f4-11ea-82ed-28c287afbf78.png">

**Tasks**: Read the text of the transcript annotations. Navigate between annotations and across pages.

**Manifest**: Book with searchable full-text annotations
- Wunder der Vererbung
https://wellcomelibrary.org/iiif/b18035723/manifest

# 5. Search
<img width="500" alt="search" src="https://user-images.githubusercontent.com/6945691/73987368-010c6b80-48f5-11ea-9ee1-f8771b2f0346.png">

**Tasks**: Use the sidebar menu to navigate to Search; enter a search term; navigate the results. 

**Manifest**: Book with searchable full-text annotations
- Wunder der Vererbung
https://wellcomelibrary.org/iiif/b18035723/manifest

# 6. Download
<img width="500" alt="download" src="https://user-images.githubusercontent.com/6945691/73988571-ee476600-48f7-11ea-9b37-41e394ae1268.png">

**Tasks**: Start from a page with 2 images. Navigate to the "Download & share" menu; select Download; select an option (page A, page B, whole object).

**Notes**: I don't think download is implemented in core Mirador, but it is in Stanford's embedded Mirador. This introduces another layer to the test:  The viewer embedded in a web page that has its own content (and potential issues). 

**Manifest**: Multi-page object that has a whole-object option.
- The songs of Mario Paci : recital [program]
https://purl.stanford.edu/yg239bh0641

# 7. Switch language
<img width="500" alt="language menu" src="https://user-images.githubusercontent.com/6945691/73987456-534d8c80-48f5-11ea-9d45-6abbdf9acc7a.png">

**Tasks**:  Navigate to the workspace sidebar and select Workspace options; select Language; select an alternate language. 

**Manifest**:  Object that has labels and/or metadata in the starting and target languages. 
- Wang, Anshi, 1021-1086: Xin ke Linchuan Wang Jiefu xian sheng shi wen ji. 2
https://api.digitale-sammlungen.de/iiif/presentation/v2/bsb00068386/manifest

**Comment by a-nnika** [comment]
Hi @camillevilla and @jvine, thanks a lot for your work, that's a really useful compilation of test screens. We will coordinate with our partner institution on when and how the testing will be done and I'll let you know as soon as I have news. Thanks!

**Comment by camillevilla** [comment]
The results of this audit made available in June 2020. [The report,](https://drive.google.com/drive/folders/19gia20X-fD_VPvVEVpTv_q4X9NYkXak9) originally written in German, is also available in auto-translated English. 

In response to this audit, [new issues](https://github.com/ProjectMirador/mirador/issues?q=is%3Aissue+sort%3Aupdated-desc+accessibility+audit+created%3A2020-06) were created with "Accessibility" tag and "Accessibility audit:" in the title. I have listed the issues below, grouped by their resolution status as of September 11, 2020.

### Open
- HTML validation? #3121
- Keyboard focus styles #3118
- Text contrast #3109
- Annotation border contrast #3113
- Text zoom overflows containers and/or breaks funny #3110

### Closed
- Sidebar <=> companion window keyboard relationship #3115
- canvas elements #3107
- <main> usage #3114
- page turning status #3123
- Sidebar toggle labeling #3117
- headings for info panel sections #3108
- provider logos overflow container at mobile screen #3111
- Catalog page #3112
- "Start Here" button aria-label #3119
- language switching #3120
